src/scss/misc/_colors.scss

```diff
@@ -18,7 +18,7 @@ $colors: (
   assets: (
     oil: (color: #729e00, icon: 'fa-database'),
     gas: (color: #ff7a28, icon: 'fa-fire'),
-    recycle: (color: #f1b71c, icon: 'fa-recycle'),
+    recycle: (color: #b11f24, icon: 'fa-recycle'),
```

Comment:
It isn't just the colors that will change though.. the shapes are also going to change by asset but changing shapes with our current map solution isn't easy as we learned yesterday. The comprehensive changes that we are going to make will come when we eventually switch to mapbox-gl but for now this is a small change for just a couple dozen wells in one state.